docs/modules/ROOT/pages/workflow-definition-from-file.adoc

```diff
@@ -0,0 +1,116 @@
+= How to load Workflow Definitions from files
+include::./includes/attributes.adoc[]
+
+This guide shows you how to load and use Serverless Workflow definitions from YAML files in your Quarkus application, making them available as CDI beans for injection and execution.
+
+
+== 1) Create the workflow specification file
+
+Create a directory `src/main/flow` in your project and add a Serverless Workflow specification file:
```

Comment:
```suggestion
Create a directory `src/main/flow` in your project and add a workflow specification file:
```

Avoid using `Serverless`, we are renaming the spec to avoid restricting our scope to this ecosystem.